docs/src/main/sphinx/release/release-471.md

```diff
@@ -0,0 +1,50 @@
+# Release 471 (19 Feb 2025)
+
+## General
+
+* Include split count and total split distribution time in the `EXPLAIN ANALYZE`
+  output. ({issue}`25028`)
+* Improve performance for queries using top N operations. ({issue}`24937`)
```

Comment:
@martint @pettyjamesm .. is this okay .. looking at the PR I thought we need to add an entry.

Reply:
The phrasing looks good to me. In practice it's probably not a very noticeable improvement in terms of end-to-end query times which is why I omitted the release notes in the PR, but no harm mentioning it.

Reply:
Exactly .. always good to mention that Trino got even faster ...